.github/workflows/pr-lint.yml

```diff
@@ -52,28 +52,32 @@ jobs:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           report-json: ./eslint-report.json
 
-      - name: Commit files
+      - name: Configure git
         run: |
           git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config --local user.name "github-actions[bot]"
 
-      - name: Prettier
-        continue-on-error: true
-        id: prettier
-        shell: bash
-        run: |
-          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
-            if [[ $file == *.js || $file == *.ts || $file == *.jsx || $file == *.tsx || $file == *.mdx || $file == *.scss || $file == *.json ]]; then
-              echo "Run Prettier on $file"
-              prettier --config .prettierrc $file --write
-            fi
-          done
-          git commit -m "chore: prettier" ${{ steps.changed-files.outputs.all_changed_files }}
-
-      - name: Push changes
-        uses: ad-m/github-push-action@master
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          branch: ${{ github.head_ref }}
+      # remove it for the moment prettier happens on pre commit
```

Comment:
Why removing it?  It introduced issues?
Pre-commit is not run if you bring changes from GitHub's Interface or using 3rd party (Supernova for example)

Reply:
IMO in some case we should be able to avoid prettier on CI side